### PR TITLE
Run go mod tidy on pkg/apis

### PR DESCRIPTION
**Pull request by ibuildthecloud**

### CODE REVIEW

File: scripts/validate [hunk 1]
```diff
@@ -17,6 +17,9 @@ echo Running: golangci-lint
 golangci-lint run
 
 echo Tidying up modules
+cd pkg/apis/
```
Comment (ibuildthecloud) [inline scripts/validate:20]: Make sure CI checks for this issue going forward

Comment (nickgerace) [reply scripts/validate:20]: Thanks for this!

**Review by nickgerace — APPROVED** [review]
Adding a committer for review

**Review by dramich — APPROVED** [review]

**Comment by nickgerace** [comment]
https://github.com/rancher/rancher/issues/34855

**Pull request merged**